src/EventSubscriber/TideSiteRouteAlter.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Drupal\tide_site\EventSubscriber;
+
+use Drupal\Core\Routing\RouteSubscriberBase;
+use Symfony\Component\Routing\RouteCollection;
+
+/**
+ * Class TideSiteRouteAlter.
+ *
+ * @package Drupal\tide_site
+ */
+class TideSiteRouteAlter extends RouteSubscriberBase {
+
+  /**
+   * Alter system.admin_content route to the view '/summary_contents'.
+   *
+   * {@inheritDoc}.
+   */
+  protected function alterRoutes(RouteCollection $collection) {
+    $route = $collection->get('system.admin_content');
+    if ($route) {
+      $collection->add('system.admin_content', clone $collection->get('view.summary_contents.page'));
```

Comment:
@vincent-gao you need to check for the existence of `view.summary_contents.page`. If `$collection->get('view.summary_contents.page')` returns NULL, cloning it will result in a fatal error.

Reply:
ok. solved. thanks.

Reply:
@vincent-gao `clone $summary_contents_route`